Austin_Marga_HW7_1.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>This spectral index is pretty meaningless. Use eg <code>np.fft.fftfreq</code>  to compute an actual frequency.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_GreenhouseGases/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>